.gitignore

```diff
@@ -37,7 +37,7 @@ src/ossec-logtest
 src/ossec-maild
 src/ossec-makelists
 src/ossec-monitord
-src/ossec-regex
```

Comment:
Let `ossec-regex` in this file along with `wazuh-regex` to avoid include it when switching between branches:

```
On branch 6782-rename-wazuh-tools
Your branch is up to date with 'origin/6782-rename-wazuh-tools'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	ossec-regex
```